_topic_map.yml

```diff
@@ -733,21 +733,21 @@ Topics:
 - Name: Administrator CLI commands
   File: administrator-cli-commands
   Distros: openshift-enterprise,openshift-origin
-#---
-#Name: Container-native Virtualization
-#Dir: cnv
-#Distros: openshift-enterprise
-#Topics:
+---
```

Comment:
Hi @ousleyp - these still need to be commented out. You can keep updating the _topic_map.yml, but make sure the CNV assemblies are commented out. We can't release them till GA.